pkg/asset/installconfig/azure/validation.go

```diff
@@ -652,3 +655,39 @@ func validateMarketplaceImage(client API, installConfig *types.InstallConfig) fi
 	}
 	return allErrs
 }
+
+func validateAzureStackDiskType(_ API, installConfig *types.InstallConfig) field.ErrorList {
+	var allErrs field.ErrorList
+
+	supportedTypes := sets.New("Premium_LRS", "Standard_LRS")
```

Comment:
https://docs.openshift.com/container-platform/4.13/installing/installing_azure_stack_hub/installing-azure-stack-hub-default.html#installation-configuration-parameters-additional-azure-stack-hub_installing-azure-stack-hub-default

Reply:
Should I add this to the commit description or maybe as a comment in the code?

Reply:
I created a bug for the docs update: https://issues.redhat.com/browse/OCPBUGS-13804